presto-native-execution/presto_cpp/main/common/Configs.cpp

```diff
@@ -290,6 +291,16 @@ std::string SystemConfig::prestoVersion() const {
   return requiredProperty(std::string(kPrestoVersion));
 }
 
+std::string SystemConfig::poolType() const {
+    static const std::unordered_set<std::string> validTypes = {"LEAF", "INTERMEDIATE", "DEFAULT"};
```

Comment:
Nit rename variable to kPoolTypes. 

Constant variable names should be prefixed with 'k'